Makefile

```diff
@@ -181,7 +183,12 @@ test-examples-kustomize-against-HEAD: $(MYGOBIN)/kustomize $(MYGOBIN)/mdrip
 test-examples-kustomize-against-latest-release: $(MYGOBIN)/mdrip
 	./hack/testExamplesAgainstKustomize.sh v5@$(LATEST_RELEASE)
 
-
+# Pushes dependencies in the go.work file back to go.mod files of each workspace module.
+.PHONY: workspace-sync
+workspace sync:
+	go mod tidy
+	go work sync
```

Comment:
Maybe that better to run `go work sync` first because to I needed rerun `go mod tidy` after `go work sync` in my experience.

Reply:
Got it, thanks!